src/components/header/LocalePicker.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { useI18n } from "vue-i18n";
+const t = useI18n();
+
+const locales = computed(() => {
+  return ["en", "pl"].map((locale) => {
+    return {
+      id: locale,
+      name: locale.toUpperCase(),
+    };
+  });
+});
+
+const handleLocaleChange = () => {
+  localStorage.setItem("LOCALE", t.locale.value);
```

Comment:
Considering this string name is re-used in multiple places, maybe we could have it refactored to a constant?
Or perhaps even getting/updating this data (the current locale) moved to a composable instead? 👀
I also seen you've put it in a pinia store, we could also use that, and have actions to update it (and save in local storage as a side effect) 😄

Reply:
I shall refactor this. I should have cleaned up better. I was planning to use pinia but give there is not really that much code there i stuck with localStorage

Reply:
In truth I am having bit of trouble with composables, I was hoping to ask you about at daily :(